src/routes/home/HomeScreen.tsx

```diff
@@ -36,13 +36,14 @@ interface HomeScreenProps {
 
 const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
   const { data: customDisciplines, refresh: refreshCustomDisciplines } = useReadCustomDisciplines()
-  const { data: selectedProfessions } = useReadSelectedProfessions()
+  const { data: selectedProfessions, refresh: refreshSelectedProfessions } = useReadSelectedProfessions()
   const isCustomDisciplineEmpty = !customDisciplines || customDisciplines.length <= 0
 
   useFocusEffect(
     React.useCallback(() => {
       refreshCustomDisciplines()
-    }, [refreshCustomDisciplines])
+      refreshSelectedProfessions()
+    }, [refreshCustomDisciplines, refreshSelectedProfessions])
   )
```

Comment:
Optional since not added in this PR: No useCallback should be needed since refreshes are already wrapped in one in the hooks.

Reply:
Without the callback we get infinite calls. `Please report: Excessive number of pending callbacks: 501. Some pending callbacks that might have leaked by never being called from native code`

Reply:
Perhaps since we have to wrap it in a new arrow function to be able to call both refreshes? Splitting it in two `useFocusEffect`s might help (did not actually test it so not sure). Anyway, fine as it is.